pegasus/test/test_asset_helpers.rb

```diff
@@ -2,26 +2,100 @@
 require_relative '../../lib/cdo/pegasus'
 require 'minitest/autorun'
 
-CDO.dashboard_assets_dir = './test/fixtures/dashboard_assets'
+# AssetMap must be re-initialized before each test with CDO.dashboard_assets_dir set properly
```

Comment:
Is this comment only relevant to development mode? Please clarify. I'm not sure what the action here is, nor what "set properly" means.